taipy/core/_scheduler/_scheduler_factory.py

```diff
@@ -17,10 +18,16 @@
 
 class _SchedulerFactory:
     @classmethod
-    def _build_scheduler(cls):
+    def _build_scheduler(cls) -> _AbstractScheduler:
         if Config.job_config._is_default_mode():
-            _Scheduler._recover_jobs()
-            return _Scheduler
-
-        package = f"taipy.{Config.job_config.mode}.scheduler"
-        return _load_fct(package, "Scheduler")
+            if util.find_spec("taipy.enterprise"):
+                # Mode standalone with enterprise version
+                package = "taipy.enterprise.core.scheduler"
```

Comment:
I think I mentioned on the new enterprise pr, I think it'd be more consistent if we have something like this: `taipy/enterprise/core/scheduler.py` will become `taipy/enterprise/core/scheduler/scheduler.py`

Reply:
yep. Good to me!